packages/demo/src/AppCCP.svelte

```diff
@@ -120,6 +82,62 @@
         "medicationStatements",
         "Sys. T",
     );
+
+    import type { LensDataPasser } from "../../lib/src/types/DataPasser";
+    let dataPasser: LensDataPasser;
+
+    window.addEventListener("emit-lens-query", (e) => {
+        const event = e as QueryEvent;
+        const { ast, updateResponse, abortController } = event.detail;
+
+        const measureItems = [
+            dktkPatientsMeasure,
+            dktkDiagnosisMeasure,
+            dktkSpecimenMeasure,
+            dktkProceduresMeasure,
+            dktkMedicationStatementsMeasure,
+            dktkHistologyMeasure,
+        ] as MeasureItem[];
+
+        const criteria = dataPasser.getCriteriaAPI("diagnosis");
+
+        const cql = translateAstToCql(
+            ast,
+            false,
+            "DKTK_STRAT_DEF_IN_INITIAL_POPULATION",
```

Comment:
Should this not come from the config? Do I need to do this for every sub project? Could we provide a interface?

Reply:
this was a showcase for using a custom backend. spots can be configured in the options to work out of the box. this code has been removed